src/LogicTasks/Helpers.hs

```diff
@@ -117,17 +118,26 @@ negationKey allowUnicode =
     pure ()
 
 arrowsKey :: OutputCapable m => LangM m
-arrowsKey = do
-  paragraph $ indent $ do
-    translate $ do
+arrowsKey = arrowsKey' [Impl, BackImpl, Equi]
+
+arrowsKey' :: OutputCapable m => [BinOp] -> LangM m
+arrowsKey' [] = pure ()
+arrowsKey' (op:os)
+  | op == Impl = go "=>" $ do
       english "Implication:"
       german "Implikation:"
-    code "=>, <="
-    pure ()
-  paragraph $ indent $ do
-    translate $ do
+  | op == BackImpl = go "<=" $ do
+      english "Back Implication:"
+      german "Back Implikation:"
```

Comment:
Ich schätze, ich fände es besser, dass es nicht getrennte Zeilen für "Implication" und "Back Implication" gibt (u.a. weil im Deutschen "Back Implikation" wirklich sehr merkwürdig klingt), sondern nach wie vor nur eine Zeile für "Implication", in der aber je nach dem nur einer oder beide der relevanten Pfeile ausgegeben werden.